src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -15,6 +15,10 @@
 use Symfony\Component\HttpKernel\DataCollector\LateDataCollectorInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Twig\Markup;
+use Twig\Profiler\Dumper\HtmlDumper;
+use Twig\Profiler\Profile;
+use Twig\Source;
```

Comment:
@keradus just wondering: this `use`is not used in the class - is it a php-cs-fixer bug that fabbot is still green?